documentation/source/users/arkane/input.rst

```diff
@@ -188,7 +189,12 @@ to apply atomization energy corrections (AEC) and spin orbit corrections (SOC) f
 (see `Model Chemistry`_). If not interested in accurate thermodynamics (e.g., if only using ``kinetics()``), then
 atom corrections can be turned off by setting ``useAtomCorrections`` to ``False``.
 
-The ``bond`` parameter is used to apply bond corrections (BC) for a given ``modelChemistry``.
+The ``bond`` parameter is used to apply bond corrections (BC) for a given ``modelChemistry`` if using Petersson-type
+bond additivity corrections (BACs) (``bondCorrectionType = 'p'``). When using Melius-type bond additivity corrections
+(``bondCorrectionType = 'm'``), specifying bonds is not required because the molecular connectivity is automatically
```

Comment:
Perhaps change `specifying bonds is not required` into `specifying bond orders is not required`

Reply:
It means that the `bonds` dictionary is not required in the input file. To highlight that, I added `` around it so that it gets parsed as plain text in the documentation.